src/Verification/VerifyAttestationWithOpenSsl.php

```diff
@@ -340,7 +335,7 @@ private function downloadAttestations(FilenameWithChecksum $file, string $owner)
                     'retry-auth-failure' => true,
                     'http' => [
                         'method' => 'GET',
-                        'header' => $this->authHelper->addAuthenticationHeader([], $this->githubApiBaseUrl, $attestationUrl),
+                        'header' => [],
```

Comment:
On second thoughts, this is affecting PIE now I'm trying to fix it there, so when I get to the bottom of it, will update this PR with the fix 😆